mixer/pkg/server/monitoring.go

```diff
@@ -61,7 +59,7 @@ func startMonitor(port uint16, enableProfiling bool, lf listenFunc) (*monitor, e
 	mux := http.NewServeMux()
 
 	registry := prometheus.NewRegistry()
-	registry.MustRegister(prometheus.NewProcessCollector(os.Getpid(), ""))
+	registry.MustRegister(prometheus.NewProcessCollector(prometheus.ProcessCollectorOpts{}))
```

Comment:
Why ? @douglas-reid

Reply:
does the client_golang update here match https://github.com/istio/istio/pull/9342 ?

Reply:
Yes. For whatever reason, Gopkg.lock was at version 0.9.0 while the Gopkg.toml constraint is 0.9.0-pre1. I updated Gopkg.toml to match #9342 and make the dep files consistent.

Reply:
`prometheus.NewProcessCollector` was [added](https://github.com/istio/istio/pull/7989/files#diff-804f2a95859b6e40df217d24109e985bR64) in the master branch when mixer self-monotiring was switched over to OpenCensus.